spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConverterAwareMappingSpannerEntityWriterTest.java

```diff
@@ -98,6 +99,11 @@ public void writeTest() {
 		t.bytesList = new ArrayList<>();
 		t.bytesList.add(t.bytes);
 
+		t.testEmbeddedColumns = new TestEmbeddedColumns();
```

Comment:
All test classes should be named \*Test**s** for consistency

Reply:
will fix.

Reply:
changes made, PTAL